exercises.md

```diff
@@ -4,36 +4,36 @@
 
 1. Clone the workshop repo and navigate to the `example` directory
 
-```
+```bash
 git clone https://github.com/Cambridge-ICCS/testing-workshop
 cd testing-workshop/exmaple
 ```
 
 2. Setup a virtual environment (optional step) and install the dependencies:
 
-```
+```bash
 python3 -m venv venv
 source venv/bin/activate
 pip install -r requirements.txt
 ```
 
 3. (Optional) To make it easy to import the Python module for exploration, set the `PYTHONPATH` to include the `src` directory:
 
-```
-export set PYTHONPATH=$PYTHONPATH:$(pwd)/src 
+```bash
+export PYTHONPATH=$PYTHONPATH:$(pwd)/src
```

Comment:
I think the `set` might have been a typo?

I also added `bash` annotations for some syntax highlighting (does not make much difference in the end though...)

Reply:
whoops - yes thanks!